.github/workflows/publish.yml

```diff
@@ -11,15 +11,13 @@ jobs:
     steps:
     - uses: actions/checkout@v1
 
-    - name: Set up Python 3.7
+    - name: Set up Python 3.10
       uses: actions/setup-python@v1
       with:
-        python-version: "3.7"
+        python-version: "3.10"
 
     - name: Build distribution
-      run: |
-        pip install setuptools_scm wheel
-        python3 setup.py sdist bdist_wheel
+      run: make build_pkg
 
     - name: Publish distribution 📦 to PyPI
       if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
```

Comment:
Warning:  You are using "pypa/gh-action-pypi-publish@master". The "master" branch of this project has been sunset and will not receive any updates, not even security bug fixes. Please, make sure to use a supported version. If you want to pin to v1 major version, use "pypa/gh-action-pypi-publish@release/v1".